src/main/java/randoop/contract/CompareToAntiSym.java

```diff
@@ -0,0 +1,67 @@
+package randoop.contract;
+
+import randoop.Globals;
+
+/**
+ * The contract: Reversing a compareTo on two objects should result in equal values of opposite signs
+ * <code>Math.signum(x0.compareTo(x1)) == -Math.signum(x1.compareTo(x0))</code>.
+ */
+public class CompareToAntiSym implements ObjectContract {
```

Comment:
Is there a reason not to write out the word rather than using the abbreviation "Sym"?